Character/CharacterStats.cpp

```diff
@@ -96,6 +96,12 @@ Equipment* CharacterStats::get_equipment() const {
     return this->equipment;
 }
 
+unsigned CharacterStats::get_block_value() const {
+  return static_cast<unsigned>(
+    round(base_stats->get_block_value() + equipment->get_stats()->get_block_value())
```

Comment:
Rounding should be unnecessary, only integers involved here. There are to my knowledge also no future mods that adds percentage increases to block value (only integer increases).